sympy/functions/elementary/trigonometric.py

```diff
@@ -397,6 +397,9 @@ def _eval_rewrite_as_sqrt(self, arg):
     def _eval_rewrite_as_csc(self, arg):
         return 1/csc(arg)
 
+    def _eval_rewrite_as_sec(self, arg):
+        return sqrt(1 - (1 / sec(arg)**2))
```

Comment:
@jksuom, Please see the [new commit](https://github.com/sympy/sympy/pull/11424/commits/28e9d2eefe784f5d66226d0a748b535dc41bc489). Now it is not using `sqrt`. Thanks.